persistence/persistence-mybatis-test/src/test/java/org/cbioportal/persistence/mybatis/PatientMyBatisRepositoryTest.java

```diff
@@ -0,0 +1,177 @@
+package org.cbioportal.persistence.mybatis;
+
+import org.cbioportal.model.CancerStudy;
+import org.cbioportal.model.Patient;
+import org.cbioportal.model.meta.BaseMeta;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Configurable;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("/testContextDatabase.xml")
+@Configurable
+public class PatientMyBatisRepositoryTest {
+    
+    @Autowired
+    private PatientMyBatisRepository patientMyBatisRepository;
+
+    @Test
+    public void getAllPatientsInStudyIdProjection() throws Exception {
+
+        List<Patient> result = patientMyBatisRepository.getAllPatientsInStudy("study_tcga_pub", "ID", null, null, null,
+            null);
+
+        Assert.assertEquals(14, result.size());
+        Patient patient = result.get(0);
```

Comment:
This was my reply https://github.com/cBioPortal/cbioportal/pull/1825#discussion_r86157639